agr_api/src/main/java/org/alliancegenome/api/rest/interfaces/CacheRESTInterface.java

```diff
@@ -25,9 +33,14 @@
 
 
     @GET
-    @Path("/{cacheSpace}/{id}")
+    @Path("/{cache_name}/{id}")
```

Comment:
Can we use camel case convention?

Reply:
Its only seen here in the API file but sure... i'll change it.